### ISSUE TITLE
does curl use the system or php timeout timeout value?

### ISSUE THREAD

**Issue by lfxc** [arodygin/php-linode-api3#3]
or can you set it in the code somewhere?

thx

**Comment by arodygin** [comment]
I have added an ability to specify additional cURL options, and released it as 1.1.10. Additional options can be (optionally) specified as a second parameter of API classes. For example:

```php
$api = new LinodeApi($key, [
    CURLOPT_CONNECTTIMEOUT => 120,    // timeout on connect in seconds
    CURLOPT_TIMEOUT        => 120,    // timeout on response in seconds
]);
```

Hope it does the trick for you.

**Comment by lfxc** [comment]
perfect!  thanks!
